src/Microsoft.AspNet.Html.Abstractions/IHtmlContent.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Framework.WebEncoders;
+
+namespace Microsoft.AspNet.Html.Abstractions
+{
+    /// <summary>
+    /// Represents a content which can write itself.
+    /// </summary>
```

Comment:
Eww 'represents' rears its ugly head again. You can take it out and the meaning is unchanged.

I'd say "HTML content which can be written to a TextWriter"